src/mem/alloc.h

```diff
@@ -1366,15 +1370,8 @@ namespace snmalloc
       large_allocator.dealloc(slab, large_class);
     }
 
-#if defined(__GNUC__) && !defined(__clang__) && !defined(__OPTIMIZE__)
-    // Don't force this to be always inlined in debug builds with GCC, because
-    // it will fail and then raise an error.
-    inline
-#else
-    SNMALLOC_FAST_PATH
-#endif
-      void
-      remote_dealloc(RemoteAllocator* target, void* p, sizeclass_t sizeclass)
+    SNMALLOC_SLOW_PATH
```

Comment:
Please add a comment explaining why moving this to the slow path is a good idea, or in 6 months someone is likely to think it should be fast path.